strr-examiner-web/app/stores/examiner.ts

```diff
@@ -42,26 +42,40 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
   })
 
   const fetchApplications = () => {
+    let statusValue = Array.isArray(tableFilters.status)
```

Comment:
Is this for multi-select?

Reply:
tableFilters.status is already `[]`. Currently the API supports only single status. I had checked with @Michaela-R-Anderson and she wasn't sure and wanted to check with Andy whether `status` and `type` should be single select or multi select.

Reply:
So, I left it unchanged pending a decision on this.